### ISSUE TITLE
ICNet evaluation mean-IOU not increasing still exist

### ISSUE THREAD

**Issue by jinfagang** [meetshah1995/pytorch-semseg#171]
Hi, I previous opened an issue about ICNet evaluation mean-iou, it seems stopped at 0.25 approximately.

```
5it [00:01,  2.34it/s]WARN: resizing labels yielded fewer classes
250it [00:23, 10.78it/s]
Overall Acc: 	 0.8265528854613082
Mean Acc : 	 0.33788360175823046
FreqW Acc : 	 0.7175463861106175
Mean IoU : 	 0.2577618935258648
Iter [199050/300000]  Loss: 0.6206  Time/Image: 0.0998
Iter [199100/300000]  Loss: 0.8367  Time/Image: 0.0995
Iter [199150/300000]  Loss: 1.2830  Time/Image: 0.0999
Iter [199200/300000]  Loss: 0.5665  Time/Image: 0.0998
Iter [199250/300000]  Loss: 0.7296  Time/Image: 0.0997
Iter [199300/300000]  Loss: 1.2682  Time/Image: 0.1001
Iter [199350/300000]  Loss: 0.8238  Time/Image: 0.0998
Iter [199400/300000]  Loss: 1.1979  Time/Image: 0.1013
Iter [199450/300000]  Loss: 0.6620  Time/Image: 0.1003
Iter [199500/300000]  Loss: 1.0504  Time/Image: 0.0996
Iter [199550/300000]  Loss: 0.6693  Time/Image: 0.1001
Iter [199600/300000]  Loss: 0.5275  Time/Image: 0.1001
Iter [199650/300000]  Loss: 0.8949  Time/Image: 0.0996
Iter [199700/300000]  Loss: 1.2642  Time/Image: 0.1001
Iter [199750/300000]  Loss: 0.7971  Time/Image: 0.1002
Iter [199800/300000]  Loss: 0.9097  Time/Image: 0.0999
Iter [199850/300000]  Loss: 1.1369  Time/Image: 0.1004

```
No matter how long I train, the result seems very bad. Here is my config about icnet:

```
model:
    arch: icnetBN
data:
    dataset: cityscapes
    train_split: train
    val_split: val
    # icnet should be 32*n+1
    img_rows: 513
    img_cols: 1025
    path: /media/jintain/sg/permanent/datasets/Cityscapes
training:
    train_iters: 300000
    batch_size: 2
    val_interval: 1000
    n_workers: 16
    print_interval: 50
    optimizer:
        name: 'sgd'
        lr: 1.0e-5
        weight_decay: 0.0005
        momentum: 0.99
    loss:
        name: 'multi_scale_cross_entropy'
        size_average: True
    lr_schedule:
#    resume: fcn8s_pascal_best_model.pkl
    resume: runs/icnet_cityscapes_best_model.pkl
    resume:

```

I am using icnetBN, and the learning rate changed twice, 1e-3 and 1e-5. my question is:

1. the accuracy seems normal, but mean-iou not, does it because calculation methods problem or something else?

**Comment by tkasarla** [comment]
learning rate mentioned in ICNet paper is 0.01.

Also there's a poly_lr scheduler. 

You will need to add those both.
